### ISSUE TITLE
Rideable is installed and enabled in my world, but it does not work, and does not show up in Configure Settings

### ISSUE THREAD

**Issue by devteamplus** [Saibot393/Rideable#19]
I just recently updated to your latest version (2.4.2) of the module.  Now, my tokens no longer have the ability to mount, unmount, or be ridden.  I removed and reinstalled the module.  However, it shows up in the main Foundry module list as not installed, even though when I click on "Install" it brings up a notice that it was installed.

Rideable does not appear in the list under "Configure Settings" either.

**Comment by Saibot393** [comment]
Could you try deleting the module folder manually (Foundry data folder>modules>Rideable) and then reinstalling it.
Also, do you know which Rideable version you had installed before the update?

**Comment by Saibot393** [comment]
Another thing, when you install it while having the console open (F12), do any errors show up?

**Comment by Saibot393** [comment]
I may have found the cause of this Problem. I have rereleased v2.4.2. with an appropiate bug fix

**Comment by devteamplus** [comment]
Sorry, had to jump into a meeting.  

So, the only thing I did was uninstall and reinstall.  And here are the results:

- The module still shows (in the main Foundry listing) as not being installed.
- Mounting and unmounting works as expected.
- Rideable appears in the "Configure Settings" options now.
- When trying to use "Find the Culprit", Rideable does NOT appear on the list of modules available to start with as enabled.

Hope this helps.  And, you may want to make this 2.4.3, so that Foundry will recognize that and update, as expected.

**Comment by Saibot393** [comment]
Ok, at least the module works now in principle, thank you for the confirmation.
The first problem is actually something that was (as far as im aware) present since v1.0.0. I have a few ideas why this is happening, but since it doesn't cause any problems and the testing for it could be quite annoying i will leave it alone for now and focus on additional features.
The "Find the Culprit" thing however is strange, especially since it shows up for me. I will leave the issue open until i find out what causes that that. I would be quite grateful if you could inform me here, should Rideable show up in "Find the Culprit" in a future update.

**Comment by devteamplus** [comment]
Sure... you bet.  I'll keep an eye on it and let you know if it returns to the list.

Thanks.

**Comment by devteamplus** [comment]
Also, thought you'd like to know that "Lock and Key" is exhibiting the same behavior in that it does not show as "installed" after a successful install.  It DOES appear in the list of modules that can be enabled/disabled.  And, it DOES appear in the list of "Configure Settings".  It DOES NOT appear in the list of modules in "Find the Culprit".

Thought you'd like to know.

**Comment by Saibot393** [comment]
Thanks for letting me know. I assume that it will be same problem, so at least i can solve both at the same time, as soon as i discover the reason for this behaviour.

**Comment by Saibot393** [comment]
I think i found the reason for both problems (at least for the showing as not installed bug). The problem is, that i use an upper case "Rideable" as the module id, while Foundry only allows for lowercase IDs for their website IDs. Therefore foundry finds "rideable" on the website, which is technically not installed, but will install "Rideable". I can not simply change my internal ID to a lowercase since that will erase all settings any user ever made in Rideable which will probably be a bit annoying for many people. There may be some way around this by changing the on website ID. The Module "Material Deck" for example did this, so maybe there is a way. For now i will leave it as it is, since the solution could bring further (more devastating) bugs.

**Comment by devteamplus** [comment]
Agreed.  Yeah, that sounds like a logical reason for the issue.  Can live with it.  They work - so no need to upset the apple cart at this point.
